src/domain/harbors/HarborsPageContainer.tsx

```diff
@@ -8,19 +8,22 @@ import { HARBORS_QUERY } from './harborsQuery';
 import Table, { Column } from '../../common/table/Table';
 import { getHarborsData, HarborData } from './utils';
 import { HARBORS } from './__generated__/HARBORS';
-import {
-  IconPole,
-  IconFence,
-  IconTrash,
-  IconWaterTap,
-} from '../../common/icons';
 import HarborDetails from './harborDetails/HarborDetails';
 import HarborsPage from './HarborsPage';
 import InternalLink from '../../common/internalLink/InternalLink';
 import styles from '../../common/icons/icon.module.scss';
+import {
+  IconExclamationCircle,
+  IconPlug,
+  IconFence,
+  IconStreetLight,
+  IconWaterTap,
+  IconTrash,
+} from '../../common/icons';
 
 export interface IconProps {
   disabled?: boolean;
+  hidden?: boolean;
```

Comment:
What is this used for?

Reply:
a workaround for the exclamation icon to hide a second outline

Reply:
But it's not applied anywhere in the component!
Can we get rid of this workaround?

Reply:
most likely, yes